app/src/main/java/org/mozilla/tv/firefox/pocket/PocketEndpoint.kt

```diff
@@ -61,9 +53,9 @@ private object PocketEndpointRaw {
 
     /** @return The global video recommendations as a raw JSON str or null on error. */
     @AnyThread // executeAndAwait hands off the request to the OkHttp dispatcher.
-    suspend fun getGlobalVideoRecommendations(version: String): String? {
+    suspend fun getGlobalVideoRecommendations(version: String, endpoint: Uri): String? {
```

Comment:
What's the different between `endpoint` and `pocketEndpoint`?

Reply:
This should probably be pocketEndpoint too, I'll change that.